spdm.rflx

```diff
@@ -0,0 +1,1487 @@
+         Slot_2_Present : Slot_Present;
+         Slot_1_Present : Slot_Present;
+         Slot_0_Present : Slot_Present;
+      end message;
+
+   -- 10.2 GET_VERSION request and VERSION response messages
+
+   type Get_Version_Request is
+      message
+         Param_1 : Reserved_8;
+         Param_2 : Reserved_8;
+      end message;
+
+   type Version_Number_Entry is
+      message
+         Alpha : Alpha
+            --  Assume a released version
+            then Update_Version_Number
+               if Alpha <= 1;
+         Update_Version_Number : Update_Version_Number;
+         Minor_Version : Minor_Version;
+         Major_Version : Major_Version;
+      end message;
+
+   type Version_Number_Entries is sequence of Version_Number_Entry;
+
+   type Version_Number_Entry_Count is range 1 .. 2**8 - 1 with Size => 8;
+
+   type Version_Response is
+      message
+         Param_1 : Reserved_8;
+         Param_2 : Reserved_8;
+         Reserved : Reserved_8;
+         Version_Number_Entry_Count : Version_Number_Entry_Count;
+         Version_Number_Entries : Version_Number_Entries
+            --  ISSUE: Componolit/RecordFlux#404
+            --  with Size => Version_Number_Entry_Count * Version_Number_Entry'Size;
+            with Size => Version_Number_Entry_Count * 16;
+      end message;
+
+   -- 10.3 GET_CAPABILITIES request and CAPABILITIES response messages
+
+   type CT_Exponent is range 0 .. 2**8 - 1 with Size => 8;
+   type PSK_Request_Cap is
+      (PSK_Req_Unsupported => 0,
+       PSK_Req_Supported   => 1)
+   with Size => 2;
+
+   --  ISSUE: Componolit/RecordFlux#609
+   --  This message differs between SPDM version 1.0 and 1.1.
+   --  Since the minor version is not accessible from
+   --  within this message one message is needed for each version
+   --  that is selected by a refinement.
+   type Get_Capabilities_Request_1_0 is
+      message
+         Param_1 : Reserved_8;
+         Param_2 : Reserved_8;
+      end message;
+
+   type Get_Capabilities_Request_1_1 is
+      message
+         Param_1 : Reserved_8;
+         Param_2 : Reserved_8;
+         Reserved_1 : Reserved_8;
+         CT_Exponent : CT_Exponent;
+         Reserved_2 : Reserved_16;
+         MAC_Cap : Boolean;
+         Encrypt_Cap : Boolean;
+         Unused_Meas_Fresh_Cap : Reserved_1;
+         Unused_Meas_Cap : Reserved_2;
+         Chal_Cap : Boolean;
+         Cert_Cap : Boolean;
+         Reserved_3 : Reserved_1;
+         Handshake_In_The_Clear_Cap : Boolean;
+         Key_Upd_Cap : Boolean;
+         H_Beat_Cap : Boolean;
+         Encap_Cap : Boolean;
+         PSK_Cap : PSK_Request_Cap;
+         Key_Ex_Cap : Boolean;
+         Mut_Auth_Cap : Boolean;
+         Pub_Key_ID_Cap : Boolean;
+         Reserved_4 : Reserved_15
+            then null
+               if --  Encrypt_Cap:
+                  --  If set, one or more of PSK_CAP or KEY_EX_CAP fields shall be specified
+                  --  accordingly to indicate support.
```

Comment:
Properly quote the standard (referencing paragraph number). Same for all other quote from the standard.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/7c1908a0db7eece9cbbb987fa14b0c9d9e4ec91f.